test/clientValidationTasks/task.py

```diff
@@ -0,0 +1,23 @@
+from CRABClient.UserUtilities import config#, getUsernameFromSiteDB
```

Comment:
since this is a crab configuration file, could it have a more meaningful name than `task.py` ? even the "abused" `crabConfiguration.py` may do

Reply:
thank you, renamed.